esx_service/cli/vmdkops_admin_test.py

```diff
@@ -288,7 +288,7 @@ def setUp(self):
         """ Setup run before each test """
         self.vol_count = 0
         self.cleanup()
-        for (datastore, url_name, path) in vmdk_utils.get_datastores():
+        for (datastore, url_name, path, url) in vmdk_utils.get_datastores():
```

Comment:
I have the same confusion. We should rename these 2 variables to clarify the difference.